frontend_tests/puppeteer_tests/admin.ts

```diff
@@ -127,6 +134,15 @@ async function test_set_invite_to_streams_policy_to_admins_only(page: Page): Pro
     await submit_stream_permissions_change(page);
 }
 
+async function test_set_invite_to_streams_policy_to_admins_and_moderators_only(
+    page: Page,
+): Promise<void> {
+    console.log("Test setting invite to streams policy to 'admins and moderators only'.");
+    await page.waitForSelector("#id_realm_invite_to_stream_policy", {visible: true});
+    await page.evaluate(() => $("#id_realm_invite_to_stream_policy").val(4).trigger("change"));
+    await submit_stream_permissions_change(page);
+}
```

Comment:
It's not a goal for puppeteer tests to test all values in a dropdown, so I'd be tempted to drop these hunks unless you had a specific reason to want them.